js/dcf-slideshow.js

```diff
@@ -107,20 +112,28 @@ class DCFSlideshow {
       let ctrlNextButton = document.createElement('button');
 
       // Add classes to slideshow controls group (Keep in DCF)
-      ctrls.classList.add('dcf-list-bare', 'dcf-btn-group', 'dcf-absolute', 'dcf-pin-right', 'dcf-pin-bottom', 'dcf-z-1');
+      ctrls.classList.add('slide-show-control',
```

Comment:
I suggest `dcf-slideshow-control`. Prefix with `dcf-`, use `slideshow-` for consistency.